requirements-opt.txt

```diff
@@ -39,3 +39,7 @@ psycopg2==2.6
 # It's a pure python module, it doesn't require anything to install,
 # but needs the pywin32 extension to work
 wmi==1.4.9
+
+# checks.d/snmp/pu
+# Require gevent
+gevent==1.0.1
```

Comment:
Why not using the last stable release, i.e [`1.0.2`](https://github.com/gevent/gevent/blob/1.0.2/changelog.rst#release-102-may-23-2015) ?